docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -113,16 +54,33 @@ image::images/kibana-fleet-settings.png[{fleet} settings]
 [role="screenshot"]
 image::images/kibana-fleet-enable.png[{fleet} showing prompt to enable central management]
 
-. On the **Agents** tab, click **Add agent** and complete the steps under
-**Enroll in {fleet}**. Skip the first step because you downloaded {agent}
-earlier.
+[discrete]
+[[add-agent-to-fleet]]
+== Step 2: Add an {agent} to {fleet}
+
+include::{beats-repo-dir}/x-pack/elastic-agent/docs/elastic-agent.asciidoc[tag=agent-install-intro]
+
+To send logs and metrics to the {stack}:
 
-. Choose an agent policy (or accept the default).
+. On the **Agents** tab, click **Add agent**, and look at the deployment
+instructions under **Enroll in {fleet}**.
 
-. Copy the command for enrolling and starting the agent.
+. Download the {agent} to your host:
 +
-The copied command contains your {kib} URL and an enrollment key that was
-generated by {fleet}.
+--
+include::{beats-repo-dir}/x-pack/elastic-agent/docs/install-elastic-agent.asciidoc[tag=install-elastic-agent]
+
+See the https://www.elastic.co/downloads/elastic-agent[download page] for other
+installation options.
+--
+
+. Under **Choose an agent policy**, notice that the default policy is selected.
+The default policy includes a system integration for collecting logs and metrics
+from the host system.
+
+. Under **Enroll and start the {agent}**, copy the install command. Notice that
+the command contains your {kib} URL and an enrollment key that was generated by
+{fleet}.
```

Comment:
Right, that was intentional because DEB and RPM distributions aren't supported for automatic upgrades. I think users will see the Linux command and use that.